requirements.txt

```diff
@@ -8,4 +8,4 @@ tqdm
 omegaconf
 
 # Quantization
-torchao-nightly==2024.3.13
+torchao-nightly==2024.3.25
```

Comment:
So we are still gonna pin to a specific version instead of using latest nightly?

Reply:
Yeah, too much risk for things to break out of our control especially now that we're public and need QLoRA consistently working/